dashboard/test/models/sections/section_test.rb

```diff
@@ -114,6 +115,24 @@ class SectionTest < ActiveSupport::TestCase
     refute student.sharing_disabled?
   end
 
+  test 'removing a student from their last section restricts sharing when under 13' do
+    section1 = Section.create @default_attrs
+    section1.sharing_disabled = true
+
+    section2 = Section.create @default_attrs
+    section2.sharing_disabled = true
+
+    student = create :student
+    student.age = 11
+    section1.add_student student
+    section2.add_student student
+
```

Comment:
The set up for this is the same as the set up for the previous test.  Any reason not to extract into some kind of before step? (Mostly just curious, not suggesting you have to).

Reply:
Since my tests modify the student, I wouldn't want to use the same one for each, so I'd have to also add some reset logic, which I didn't feel was worth it. I could use the same section, but I didn't want to modify the share setting for the global `@section`